modules/installation-configuration-parameters.adoc

```diff
@@ -526,6 +526,10 @@ accounts for the dramatically decreased machine performance.
 |The number of compute machines, which are also known as worker machines, to provision.
 |A positive integer greater than or equal to `2`. The default value is `3`.
 
+|`featureSet`
+|Enables the cluster for one or more feature sets. A feature set is a collection of {product-title} features that are not enabled by default. For more information about enabling a feature set during installation, see "Enabling features using feature gates".
+|A list of feature set names as strings, such as `TechPreviewNoUpgrade`.
```

Comment:
Only one feature set at a time.

It's not a list, just a string.

Reply:
Even though we do not expect most users to do so, can you set more than one feature set at a time during installation? If so, would it be:

`featureSet: abc`
`featureSet: xyz`

I am thinking more along the lines for internal folks who might need to test or develop with more than one feature set enabled.

Reply:
No, the API only allows you to set one feature set at a time.